inputs/orchestrator_inner:1.json

```diff
@@ -35,6 +35,13 @@
   "prepublish_plugins": [],
   "exit_plugins": [
     {
+      "name": "store_metadata_in_osv3",
+      "args": {
+        "url": "{{OPENSHIFT_URI}}",
+        "verify_ssl": false
+      }
+    },
+    {
```

Comment:
Added a separate commit for adding test coverage to ensure plugins are optional.